python/lsst/daf/butler/formatters/parquet.py

```diff
@@ -111,21 +114,36 @@ def _standardizeColumnParameter(self, columns: Dict[str, Union[str, List[str]]])
         name : `str`
             Stringified tuple representing a multi-index column name.
         """
-        if not isinstance(columns, collections.abc.Mapping):
-            raise ValueError("columns parameter for multi-index data frame must be a dictionary.")
-        if not set(self.indexLevelNames).issuperset(columns.keys()):
-            raise ValueError(f"Cannot use dict with keys {set(columns.keys())} "
-                             f"to select columns from {self.indexLevelNames}.")
-        factors = [iterable(columns.get(level, self.columns.levels[i]))
-                   for i, level in enumerate(self.indexLevelNames)]
-        for requested in itertools.product(*factors):
-            for i, value in enumerate(requested):
-                if value not in self.columns.levels[i]:
-                    raise ValueError(f"Unrecognized value {value!r} for index {self.indexLevelNames[i]!r}.")
-            yield str(requested)
-
-    def read(self, columns: Union[str, List[str], Dict[str, Union[str, List[str]]]] = None
-             ) -> pd.DataFrame:
+        if isinstance(columns, list):
+            for requested in columns:
+                if not isinstance(requested, tuple):
+                    raise ValueError(
+                        "columns parameter for multi-index data frame must be either a dictionary or list of tuples."  # noqa
```

Comment:
What is this bare noqa doing?

Reply:
I thought the flake8 was failing because of a long line here, and thought this might fix it?

Reply:
If the line is too long, split the string in two.

```python
raise ValueError("columns parameter for multi-index data frame "
                 "must be either a dictionary or list of tuples.")
```

Reply:
Is this an OK thing to do?  Is there a better way to do this?

Reply:
Splitting strings like this is extremely common. It's used all over the place.